packages/browser/test/unit/tracing/browserTracingIntegration.test.ts

```
@@ -714,6 +714,45 @@ describe('browserTracingIntegration', () => {
         },
       });
     });
+
+    it("saves the span's negative sampling decision and its DSC on the propagationContext when the span finishes", () => {
```

just thought I'd throw in a semi-related test that was previously missing